packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -3142,6 +3390,131 @@ curl -v -X POST \
 }
 ```
 
+### Verify WebAuthn Factor
+
+<ApiOperation method="post" url="/api/v1/users/${userId}/factors/${factorId}/verify" />
+
+Verifies a user with a WebAuthn factor. The verification process starts with getting the WebAuthn credential request options, which are used to help select an appropriate authenticator using the WebAuthn API.
+This authenticator then generates an assertion which may be used to verify the user.
```

Comment:
need a comma before which:

This authenticator then generates an assertion, which may be used to verify the user.